src/helpers/get-ogc-items-stream.ts

```diff
@@ -0,0 +1,25 @@
+import { PagingStream } from '../paging-stream';
+import { getPagingStream } from './get-paging-stream';
+import * as _ from 'lodash';
+export const getOgcItemsStream = 
+  async (siteUrl: string, collectionKey: string, requestQuery: Record<string, any>): Promise<PagingStream> => {
+  const requestUrl = buildSearchRequestUrl(siteUrl, collectionKey, requestQuery);
+  // OGC Search API can only return 100 results at maximum if limit is 
+  // provided in the request query and the limit is over 100.
+  // So, page limit is set to 1 regardless of the actual
+  // value of limit provided in the request query
+  const maxPageLimit = requestQuery.limit && 1;
```

Comment:
The code comment isn't clear enough for me at least. `maxPageLimit` is undefined if `limit` is undefined, but 1 in all other cases; why?  And is there a difference between `limit` and "page-limit"?